src/appleseed/foundation/image/colorspace.h

```diff
@@ -236,10 +237,19 @@ Color<T, 3> linear_rgb_to_hsl(const Color<T, 3>& linear_rgb);
 template <typename T>
 Color<T, 3> ciexyz_to_linear_rgb(const Color<T, 3>& xyz);
 
+template <typename T>
+Color<T, 3> ciexyz_to_linear_rgb(
```

Comment:
@dictoon i would just name them CIEXYZ_to_RGB. CIE xyz != CIE XYZ. And the transformation implies linear RGB values anyway. As it is, ciexyz_to_linear_rgb means one needs to read the code to see if we're referring to CIE xyz or XYZ. It's not clear, just my .02c.